python/lang/security/dangerous-code-run.yaml

```diff
@@ -0,0 +1,132 @@
+rules:
+  - id: dangerous-interactive-code-run
+    mode: taint
+    options:
+      symbolic_propagation: true
+    pattern-sources:
+      - patterns:
+          - pattern-either:
+              - pattern: flask.request.$ANYTHING
```

Comment:
This could be problematic you may want to scope down only addressable things from request e.g. ARGS. headers .body etc.

Reply:
Fixed in 95b335ce1be919443c93844add4872a675e0be57